### ISSUE TITLE
Conversion of EVAL(Extender) truncates the extender value 

### ISSUE THREAD

**Issue by m-tyler** [BrianGarland/vscode-rpgfree#101]
This is the before code
```
     C                   EVAL(H)   PHAMT# = PHHRS * PHRAT#
     C                   EVAL(M)   PHAMT# = PHHRS * PHRAT#
     C                   EVAL(R)   PHAMT# = PHHRS * PHRAT#
```
This is the conversion
```
       PHAMT# = PHHRS * PHRAT#;
       PHAMT# = PHHRS * PHRAT#;
       PHAMT# = PHHRS * PHRAT#;
```
Expected output
```
       EVAL(H) PHAMT# = PHHRS * PHRAT#;
       EVAL(M) PHAMT# = PHHRS * PHRAT#;
       EVAL(R) PHAMT# = PHHRS * PHRAT#;
```
Others like CHAIN(N), DUMP(A) seem to work

**Comment by m-tyler** [comment]
Fixed format MULTI(H), DIV(H), ADD(H), and SUB(H) suffer from this issue also.  
Before
```
     C     PCT(XIDX)     MULT(H)   DDAMT#        DEDAMT
     C     PCT(XIDX)     DIV(H)    DDAMT#        DEDAMT
     C     PCT(XIDX)     ADD(H)    DDAMT#        DEDAMT
     C     PCT(XIDX)     SUB(H)    DDAMT#        DEDAMT
```
After
```
    Eval(H) DEDAMT = PCT(XIDX) * DDAMT#;
    Eval(H) DEDAMT = PCT(XIDX) / DDAMT#; 
    Eval(H) DEDAMT = PCT(XIDX) + DDAMT#;
    Eval(H) DEDAMT  = PCT(XIDX) - DDAMT#;
```

**Comment by m-tyler** [comment]
ADD SQRT, Z-ADD, and Z-SUB to the list.

**Comment by BrianGarland** [comment]
The extender is now checked when converting EVAL, ADD, SUB, MULT, DIV, SQRT, Z-ADD, & Z-SUB.

Test cases added for these scenarios. 

Fixed in version 0.0.29

**Comment by m-tyler** [comment]
Updated extension!  Thanks.
